core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -99,6 +98,10 @@ class SubmissionMonitorActor(val workspaceName: WorkspaceName,
       logger.debug(s"check current workflow status counts for submission $submissionId")
       checkCurrentWorkflowStatusCounts(true) pipeTo parent
 
+    case Status.Failure(_: SubmissionDeletedException) =>
+      logger.debug(s"submission $submissionId has been deleted, terminating disgracefully")
+      stop(self)
```

Comment:
We can't call `checkCurrentWorkflowStatusCounts` because that tries to get a bunch of info from a workspace that doesn't exist any more.